app/src/main/res/values/strings.xml

```diff
@@ -294,8 +294,7 @@
     <string name="pref_next_up_timeout_summary">How long to wait before playing the next item</string>
     <string name="pref_next_up_timeout_title">Next Up timer duration</string>
     <string name="watch_now">Watch now</string>
-    <string name="pref_next_up_enabled_summary">Previews the next queued item before playing</string>
-    <string name="pref_next_up_enabled_title">Enable Next Up screen</string>
+    <string name="pref_next_up_behavior_title">Show Next Up screen</string>
```

Comment:
```suggestion
    <string name="pref_next_up_behavior_title">Show next video info</string>
```

Stolen from jellyfin-web:

![image](https://user-images.githubusercontent.com/2305178/121952074-0f299200-cd5c-11eb-881c-303c5dc049f3.png)

Reply:
Planning for #922, would something like this make sense?

Show next video info

- After playback, with thumbnail
- After playback, without thumbnail
- During playback
- Never

With whichever one ends up being the default value as the first option listed (not sure if the plan is to go back to during playback as default). The timer duration could also stay as positive with these strings.

Reply:
The title would just be "Show next video info" so it doesn't indicate when it shows up.